base/commands/migration/migration_status.go

```diff
@@ -0,0 +1,60 @@
+//go:build std || migration
+
+package migration
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hazelcast/hazelcast-commandline-client/clc/ux/stage"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/plug"
+)
+
+type StatusCmd struct{}
+
+func (s StatusCmd) Unwrappable() {}
+
+func (s StatusCmd) Init(cc plug.InitContext) error {
+	cc.SetCommandUsage("status")
+	cc.SetCommandGroup("migration")
+	help := "Get status of the data migration in progress"
+	cc.AddStringFlag(flagOutputDir, "o", "", false, "output directory for the migration report, if not given current directory is used")
+	cc.SetCommandHelp(help, help)
+	return nil
+}
+
+func (s StatusCmd) Exec(ctx context.Context, ec plug.ExecContext) error {
+	ci, err := ec.ClientInternal(ctx)
+	if err != nil {
+		return err
+	}
+	ec.PrintlnUnnecessary("")
+	ec.PrintlnUnnecessary(banner)
+	sts := NewStatusStages()
+	sp := stage.NewFixedProvider(sts.Build(ctx, ec)...)
+	mID, err := stage.Execute(ctx, ec, any(nil), sp)
+	if err != nil {
+		return err
+	}
+	mStages, err := createMigrationStages(ctx, ec, ci, mID.(string))
+	if err != nil {
+		return err
+	}
+	mp := stage.NewFixedProvider(mStages...)
+	_, err = stage.Execute(ctx, ec, any(nil), mp)
+	err2 := finalizeMigration(ctx, ec, ci, mID.(string), ec.Props().GetString(flagOutputDir))
```

Comment:
Should finalizeMigration be called even if stage.Execute panics?

Reply:
I think it should be called when there is an error executing stage.Execute. Member logs and migration report should be saved.